internal/consumergroups/consumer-group-operation.go

```diff
@@ -173,14 +177,30 @@ func (operation *ConsumerGroupOperation) DescribeConsumerGroup(flags DescribeCon
 			consumerGroupDescription.Members = nil
 		} else if flags.OutputFormat == "wide" || flags.OutputFormat == "" {
 			tableWriter := output.CreateTableWriter()
-			if err := tableWriter.WriteHeader("CLIENT_HOST", "CLIENT_ID", "TOPIC", "ASSIGNED_PARTITIONS"); err != nil {
+
+			columns := make([]string, 0, 5)
+			columns = append(columns, "CLIENT_HOST", "CLIENT_ID")
+			if supportsGroupInstanceID {
+				columns = append(columns, "GROUP_INSTANCE_ID")
```

Comment:
I would prefer always showing the `GROUP_INSTANCE_ID` column to keep the logic simple here.
If it is not supported there will just be an empty column which does not bother much and eventually hopefully everyone will have a kafka cluster that supports it.